### ISSUE TITLE
Release failed on bnd baseline maven plugin

### ISSUE THREAD

**Issue by antoinesd** [eclipse/microprofile-health#237]
Running release on CI fails with the following message:

```
[INFO] [INFO] --- bnd-baseline-maven-plugin:4.2.0:baseline (baseline) @ microprofile-health-api ---
[INFO] [INFO] Determining the baseline version for org.eclipse.microprofile.health:microprofile-health-api:jar:(,2.2-RC2) using repositories [ossrh (https://oss.sonatype.org/service/local/staging/deploy/maven2/, default, releases+snapshots), eclipse.maven.central.mirror (https://repo.eclipse.org/content/repositories/maven_central/, default, releases)]
[INFO] [WARNING] Could not transfer metadata org.eclipse.microprofile.health:microprofile-health-api/maven-metadata.xml from/to ossrh (https://oss.sonatype.org/service/local/staging/deploy/maven2/): Failed to transfer file https://oss.sonatype.org/service/local/staging/deploy/maven2/org/eclipse/microprofile/health/microprofile-health-api/maven-metadata.xml with status code 400
[INFO] [INFO] The baseline version was found to be 2.2-RC1
[INFO] [ERROR] Baseline mismatch for package org.eclipse.microprofile.health, MAJOR change. Current is 1.2.0, repo is 1.2.0, suggest 2.0.0 or -
[INFO] [ERROR] The bundle version change (2.2.0 to 2.2.0) is too low, the new version must be at least 3.0.0 
```

**Comment by antoinesd** [comment]
As it worked last week, issue seems to come from modification in build introduced in #228. Any idea @Emily-Jiang ?

**Comment by Emily-Jiang** [comment]
Since the released version pushed to maven central, the baseline is now 2.2-RC1. Let me pull in and work out what changes are added. It suggests major changes were added since RC1.

**Comment by Emily-Jiang** [comment]
ah. Let's change the baseline to 2.1, which should fix this as we made some deletion between RCx releases.

**Comment by Emily-Jiang** [comment]
I'll do a PR shortly.

**Comment by Emily-Jiang** [comment]
I have done a PR to move the baseline to previous final release so that you can delete methods between RCx.

**Comment by Emily-Jiang** [comment]
Closes #238

**Comment by Emily-Jiang** [comment]
Close this as the build is back to normal.
